src/test/rpc/AccountObjects_test.cpp

```diff
@@ -1220,6 +1220,142 @@ class AccountObjects_test : public beast::unit_test::suite
         }
     }
 
+    void
+    testAccountObjectMarker()
+    {
+        testcase("AccountObjectMarker");
+
+        using namespace jtx;
+        Env env(*this);
+
+        Account const alice{"alice"};
+        Account const bob{"bob"};
+        Account const carol{"carol"};
+        env.fund(XRP(10000), alice, bob, carol);
+
+        unsigned const accountObjectSize = 30;
+        for (unsigned i = 0; i < accountObjectSize; i++)
+            env(check::create(alice, bob, XRP(10)));
+
+        for (unsigned i = 0; i < 10; i++)
+            env(token::mint(carol, 0));
+
+        env.close();
+
+        unsigned const limit = 11;
+        Json::Value marker;
+
+        // test account_objects with a limit and update marker
+        {
+            Json::Value params;
+            params[jss::account] = bob.human();
+            params[jss::limit] = limit;
+            params[jss::ledger_index] = "validated";
+            auto resp = env.rpc("json", "account_objects", to_string(params));
+            auto& accountObjects = resp[jss::result][jss::account_objects];
+            marker = resp[jss::result][jss::marker];
+            BEAST_EXPECT(!resp[jss::result].isMember(jss::error));
+            BEAST_EXPECT(accountObjects.size() == limit);
+        }
+
+        // test invalid marker by adding invalid string after the maker.
+        {
+            std::string s = marker.asString();
+            s.append(",1234");
```

Comment:
I suggest replacing this test with a different one, where `entryIndex` is valid, see my other comment.

Reply:
This is used to test adding some extra characters after the entryIndex. The marker format is like `xxxx,yyyy,1234`.

Reply:
Nice, I missed it.